dev/ColorPicker/ColorPicker.xaml

```diff
@@ -254,9 +145,81 @@
                             </VisualStateGroup>
+                            <VisualStateGroup x:Name="Orientation">
+                                <VisualState x:Name="Horizontal">
+                                    <VisualState.Setters>
+                                        <Setter Target="RootGrid.MinHeight" Value="{StaticResource ColorPickerVerticalOrientationMinHeight}"/>
+                                        <Setter Target="RootGrid.MaxHeight" Value="{StaticResource ColorPickerVerticalOrientationMaxHeight}"/>
+
+                                        <Setter Target="ThirdDimensionSliderGrid.(Grid.Column)" Value="1"/>
+                                        <Setter Target="ThirdDimensionSlider.Orientation" Value="Vertical"/>
+                                        <Setter Target="ThirdDimensionBackgroundRectangle.Width" Value="11"/>
+                                        <Setter Target="ThirdDimensionBackgroundRectangle.VerticalAlignment" Value="Stretch"/>
+                                        <Setter Target="ThirdDimensionBackgroundRectangle.HorizontalAlignment" Value="Center"/>
+                                        <Setter Target="ThirdDimensionBackgroundRectangle.RenderTransform">
+                                            <Setter.Value>
+                                                <RotateTransform Angle="180"/>
+                                            </Setter.Value>
+                                        </Setter>
+
+                                        <Setter Target="AlphaSliderGrid.(Grid.Column)" Value="2"/>
+                                        <Setter Target="AlphaSlider.Orientation" Value="Vertical"/>
+                                        <Setter Target="AlphaSliderCheckeredBackgroundRectangle.Width" Value="11"/>
+                                        <Setter Target="AlphaSliderCheckeredBackgroundRectangle.HorizontalAlignment" Value="Center"/>
+                                        <Setter Target="AlphaSliderGradientBackgroundRectangle.Width" Value="11"/>
+                                        <Setter Target="AlphaSliderGradientBackgroundRectangle.HorizontalAlignment" Value="Center"/>
+                                        <Setter Target="AlphaSliderGradientBackgroundRectangle.RenderTransform">
+                                            <Setter.Value>
+                                                <RotateTransform Angle="180"/>
+                                            </Setter.Value>
+                                        </Setter>
+                                    
+                                        <Setter Target="MoreEntriesPanel.(Grid.Column)" Value="3"/>
+                                        <Setter Target="MoreButton.Margin" Value="0,0,0,12"/>
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Vertical">
```

Comment:
There should always be one orientation that is the default visual state, and not require any additional setters.

Reply:
Is that beneficial here? This would mean that we would need to introduce setters whose whole purpose is to undo the "default" settings (e.g. Grid.Row).

Reply:
Yes. This will make it more readable as well.

Reply:
Updated the code now :)